services/pane.js

```diff
@@ -215,56 +225,59 @@ function addPreview(preview) {
 
 /**
  * format and assemble error messages
- * @param {Object[]} errors
+ * @param {Object[]} errors (or warnings)
+ * @param {string} [modifier] modifier class for warnings, info, etc
  * @returns {Element}
  */
-function addErrors(errors) {
+function addErrors(errors, modifier) {
   return _.reduce(errors, function (el, error) {
     var errorEl = exports.getTemplate('.publish-errors-template'),
       errorLabel = dom.find(errorEl, '.label'),
       errorDescription = dom.find(errorEl, '.description'),
       list = dom.find(errorEl, '.errors');
 
-    // reset template label default if available
+    // add rule label if it exists
     if (errorLabel && _.get(error, 'rule.label')) {
       errorLabel.innerHTML = error.rule.label + ':';
     }
 
-    // reset template description default if available
+    // add rule description if it exists
     if (errorDescription && _.get(error, 'rule.description')) {
       errorDescription.innerHTML = error.rule.description;
     }
 
-    // add each place where the error occurs
+    // add each place where the error/warning occurs
     _.each(error.errors, function (item) {
       var itemEl = dom.create(`<li><span class="error-label">${item.label}</span>${addPreview(item.preview)}</li>`);
 
       list.appendChild(itemEl);
     });
 
     el.appendChild(errorEl);
+    // add modifier class if it exists
+    if (modifier) {
+      dom.find(el, '.publish-error').classList.add(modifier);
+    }
     return el;
   }, document.createDocumentFragment());
 }
 
 /**
  * open validation error pane
- * @param {Object[]} errors
- * @param {object} errors[].rule
- * @param {string} errors[].rule.label e.g. 'Required'
- * @param {string} errors[].rule.description e.g. 'Required fields cannot be blank'
- * @param {Object[]} errors[].errors
- * @param {string} errors[].errors[].label e.g. 'Article > Header'
- * @param {string} [errors[].errors[].preview] e.g. 'text in a paragraph TK more text...'
+ * @param {object} validation
+ * @param {Object[]} validation.errors
+ * @param {Object[]} validation.warnings
  */
-function openValidationErrors(errors) {
+function openValidationErrors(validation) {
```

Comment:
This function is being used for both errors and warnings, right? If it is – would be nice to have it named something more generic like `openValidationMessages()`.